package.json

```diff
@@ -13,6 +13,7 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
+    "htm": "^3.1.1",
```

Comment:
Yes, same mechanism (although with JSX instead of htm):
https://github.com/bpmn-io/bpmn-js-properties-panel/blob/cb627c4e2c21aef5f37c4723f2cdb219a66ee310/src/contextProvider/zeebe/DescriptionProvider.js#L27-L29

Reply:
Any reason we chose a different mechanism?

Reply:
We are currently not bundeling this package, just publish as-is to npm. With JSX, we would need to add a bundling step (+ resolve to correct Preact location). This was just easier to do.

`@bpmn-io/diagram-js-ui` already uses `htm` as well, so it would not increase the bundle size of `camunda-bpmn-js` either